airbyte-integrations/connectors/source-google-pagespeed-insights/source_google_pagespeed_insights/google_pagespeed_insights.yaml

```diff
@@ -0,0 +1,63 @@
+version: "0.1.0"
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://www.googleapis.com/pagespeedonline/v5/"
+    http_method: "GET"
+    request_options_provider:
+      request_parameters:
+        key: "{{ config['api_key'] }}"
+        url: "{{ config['url'] }}"
+        category: seo
+        cateogry: accessibility
+        cateogry: best-practices
+        cateogry: performance
+        cateogry: pwa
+        strategy: desktop
+        strategy: mobile
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  pagespeed_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "pagespeed"
+      path: "/runPagespeed"
+  origin_pagespeed_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "origin_pagespeed"
+      path: "/runPagespeed"
+    retriever:
+      $ref: "*ref(definitions.base_stream.retriever)"
+      requester:
+        url_base: "https://www.googleapis.com/pagespeedonline/v5/"
+        http_method: "GET"
+        request_options_provider:
+          request_parameters:
+            key: "{{ config['api_key'] }}"
+            url: "origin:{{ config['url'] }}"
+            category: seo
+            cateogry: accessibility
+            cateogry: best-practices
+            cateogry: performance
+            cateogry: pwa
+            strategy: desktop
+            strategy: mobile
```

Comment:
After testing this a bit, it looks like the API only accepts one of these at the same time, not both. When both are added, the numbers seem to default to the `DESKTOP` option.

I think we should make this into a config option. I think we should also include in the record which strategy we were using, as the API does not seem to return any indication of the one used.